File: day 54/homework/hm.py
# User Input Number Division
# Ask the user to enter two numbers and divide them. Handle errors like division by zero and non-numeric input.



try:
    User=int(input("Enter number: "))
    User1=int(input("Enter number: "))
    print(User/User1)
except ZeroDivisionError:
    print("your second number can't be zero")
except ValueError:
    print("you can't input non-numeric input")

File: day 69/classwork/hm.js
function discountFunc(){
    let User=Number(prompt("Enter your age: "))

    if (User<18){
        console.log("discount is 20%")

    }

    else if (User>=18 && User<65){
        console.log("discount is 5%")
    }
    else{
        console.log("discount is 10%")
    }

}